### PR TITLE
42 login and registration forms

### LINKED ISSUE #42: login and registration forms

### PR DESCRIPTION
**Pull request by Ruslanchik01**
closed #42

### CODE REVIEW

**Review by daniilievtukhov — APPROVED** [review]

File: src/components/Layout/AuthModal/components/Register/Register.jsx [hunk 1]
```diff
@@ -22,14 +22,187 @@ const advices = [
 ];
 
 export function Register() {
+
+	const [firstName, SetFirstName] = useState({name:"", error:"", state:true});
```
Comment (IhorKopieichykov) [inline src/components/Layout/AuthModal/components/Register/Register.jsx]: why is the name of the setter capitalized?
```suggestion
	const [firstName, setFirstName] = useState({name:"", error:"", state:true});
```

File: src/components/Layout/AuthModal/components/Register/Register.jsx [hunk 2]
```diff
@@ -22,14 +22,187 @@ const advices = [
...
+	const [lastName, SetLastName] = useState({name:"", error:"", state:true});
```
Comment (IhorKopieichykov) [inline src/components/Layout/AuthModal/components/Register/Register.jsx]: same thing
```suggestion
	const [lastName, setLastName] = useState({name:"", error:"", state:true});
```

File: src/components/Layout/AuthModal/components/Register/Register.module.scss [hunk 3]
```diff
@@ -3,35 +3,80 @@
...
+        min-width: 250px;
+        font-size: 16px;
+        line-height: 16px;
+        border-radius: 8px;
+    }
+    & label{
+        font-size: 14px;
+        font-weight: 400;
+        padding: 0 5px;
+        position: absolute;
```
Comment (IhorKopieichykov) [inline src/components/Layout/AuthModal/components/Register/Register.module.scss]: it is desirable to specify the exact position of an element when using position absolute
Use props "left", "top" or "right", "bottom"
On the other hand, the way you place this label doesnt require "position: absolute" at all :/

**Review by IhorKopieichykov — APPROVED** [review]
gg wp

**Pull request merged**